packages/xo-server/src/xo-mixins/resource-sets.js

```diff
@@ -124,7 +124,7 @@ export default class {
 
   async computeVmResourcesUsage(vm) {
     return Object.assign(
-      computeVmResourcesUsage(this._xo.getXapi(vm).getObject(vm._xapiId)),
+      computeVmServerResourcesUsage(vm),
```

Comment:
`computeVmServerResourcesUsage(this._xo.getXapi(vm).getObject(vm._xapiId))`

Reply:
As i see, this method only needs the VM `xapiId`. So, it makes sense to pass directly an ID instead of an object. This practice is done for public methods, private methods can take objects.

Reply:
No, you were right, I only meant to change the name in this commit, I'll fix it.